jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -263,6 +281,8 @@ private CookieManager newCookieManager()
     @Override
     protected void doStop() throws Exception
     {
+        LifeCycle.stop(destinationIdleTimeoutSweeper);
```

Comment:
To avoid exception wrapping, this should be:

```java
if (destinationIdleTimeoutSweeper != null)
    destinationIdleTimeoutSweeper.stop();
```

Reply:
Done.